bindata/manifests/operator-webhook/server.yaml

```diff
@@ -32,13 +32,21 @@ spec:
           requiredDuringSchedulingIgnoredDuringExecution:
             nodeSelectorTerms:
             - matchExpressions:
+              {{ if .ExternalControlPlane }}
```

Comment:
in case of external control plane maybe drop the entire  'requiredDuringScheduling....' 

or alternatively just add another `- matchExpression` as these are ORed

Reply:
dropped `requiredDuringScheduling` in External mode.